guides/doc-Provisioning_Guide/topics/proc_configuring_iPXE_to_reduce_provisioning_times.adoc

```diff
@@ -36,7 +36,14 @@ There are three methods of using iPXE with {ProjectName}:
 
 .Security Information
 
-The iPXE binary in {RHEL} is built without some security features. For this reason, you can use HTTP, and cannot use HTTPs. All security-related features of iPXE in {RHEL} are not supported. For more information, see https://access.redhat.com/solutions/3483601[Red Hat Enterprise Linux HTTPS support in iPXE].
+The iPXE binary in {RHEL} is built without some security features. For this reason, you can use HTTP, and cannot use HTTPs.
++
+ifeval::["{build}" != "satellite"]
+Recompile iPXE from source to use security features like HTTPs.
```

Comment:
```suggestion
Recompile iPXE from source to use security features like HTTPS.
```

Reply:
@lzap, please commit the suggestion and I will merge.